msteams-platform/apps-in-teams-meetings/build-apps-for-anonymous-user.md

```diff
@@ -9,13 +9,34 @@ ms.localizationpriority: medium
 
 # Build apps for anonymous users
 
-You can build bots, messaging extensions, cards and task modules in your app to engage with anonymous meeting participants.
+You can build tabs and task modules in your app to engage with anonymous meeting participants.
+
+To build apps for anonymous users ensure the following:
+
+* [Enable `supportsAnonymousGuestUsers` to true in your app manifest](#manifest-update-for-supportsanonymousguestusers).
+* [Admin setting for anonymous user app interaction](#admin-setting-for-anonymous-user-app-interaction).
+
+## Manifest update for `supportsAnonymousGuestUsers`
+
+To allow anonymous users to interact with the tab app, ensure to update your app manifest `meetingExtensionDefinition` property with `supportsAnonymousGuestUsers` to `true`. Following is an example of the manifest:
+
+```json
+
+ "meetingExtensionDefinition": {
+  "supportsAnonymousGuestUsers": true
+ }
+
+```
 
 To test your app's experience for anonymous users, select the URL in the meeting invite and join the meeting from a private browser window.
 
+## Anonymous user authentication flow
+
+As anonymous users aren't AAD accounts, skype token isn't available for anonymous users in third-party apps. This means that if `getAuthToken` is called in client SDK it would fail. Ensure that your app identifies the anonymous users and provides the correct experience in the meetings. You can use [Get user context](~/tabs/how-to/access-teams-context.md#get-context-by-using-the-microsoft-teams-javascript-library) API to determine whether user is anonymous or not and redirect to the SSO page of the app.
+
 ## Admin setting for anonymous user app interaction
```

Comment:
On line 16/17 this is listed as a requirement along with "## Manifest update for `supportsAnonymousGuestUsers`". Can we move this directly below that section to avoid splitting the content with a paragraph on "## Anonymous user authentication flow". Or should "Anonymous user authentication flow" be listed as a requirement on L16/17 too?

Reply:
Moved the section below content